common/app/model/dotcomrendering/pageElements/EditionsCrosswordRenderingDataModel.scala

```diff
@@ -1,33 +1,15 @@
 package model.dotcomrendering.pageElements
-
-import com.gu.contentapi.client.model.v1.Crossword
-import com.gu.contentapi.json.CirceEncoders._
-import io.circe.syntax._
-import implicits.Dates.CapiRichDateTime
-import model.dotcomrendering.DotcomRenderingUtils
-import play.api.libs.json.{JsObject, Json, JsValue}
+import model.CrosswordData
+import play.api.libs.json.{Json, JsValue}
 
 case class EditionsCrosswordRenderingDataModel(
-    crosswords: Iterable[Crossword],
+    crosswords: Iterable[CrosswordData],
 )
 
 object EditionsCrosswordRenderingDataModel {
-  def apply(crosswords: Iterable[Crossword]): EditionsCrosswordRenderingDataModel =
-    new EditionsCrosswordRenderingDataModel(crosswords.map(crossword => {
-      val shipSolutions =
-        crossword.dateSolutionAvailable
-          .map(_.toJoda.isBeforeNow)
-          .getOrElse(crossword.solutionAvailable)
-
-      if (shipSolutions) {
-        crossword
-      } else {
-        crossword.copy(entries = crossword.entries.map(_.copy(solution = None)))
-      }
-    }))
-
-  def toJson(model: EditionsCrosswordRenderingDataModel): JsValue =
+  def toJson(model: EditionsCrosswordRenderingDataModel): JsValue = {
     Json.obj(
-      "crosswords" -> Json.parse(model.crosswords.asJson.deepDropNullValues.toString()),
+      "crosswords" -> Json.toJson(model.crosswords),
```

Comment:
If we were to instead pass the new data alongside here:

```suggestion
      "crosswords" -> model.crosswords.asJson.deepDropNullValues.toString(),
      "newCrosswords" -> Json.toJson(model.crosswords),
```

we could swap DCAR over from one to the other, and then come back and tidy this up?